ooni/report/cli.py

```diff
@@ -20,7 +20,7 @@ class Options(usage.Options):
          "Specify the configuration file to use."],
         ["collector", "c", None,
          "Specify the collector to upload the result to."],
-        ["bouncer", "b", "httpo://nkvphnp3p6agi5qq.onion",
+        ["bouncer", "b", None,
```

Comment:
Yes this is true, but it was actually a bug in the previous version of oonireport. By having the default set to the canonical bouncer meant that if somebody used oonireport to upload incomplete reports that had been originally submitted to a custom collector (for example because they ran `ooniprobe -c httpo://mycustomcollector`) instead of submitting them to that collector we would be submitting them to our collector.